React/Base/RCTUtils.h

```diff
@@ -94,11 +94,13 @@ RCT_EXTERN BOOL RCTForceTouchAvailable(void);
 
 // Return a UIAlertView initialized with the given values
 // or nil if running in an app extension
+#if !TARGET_OS_TV
 RCT_EXTERN UIAlertView *__nullable RCTAlertView(NSString *title,
```

Comment:
Since you've removed all call-sites to this method, can you just remove this method completely?